conditionalSwitch.js

```diff
@@ -0,0 +1,14 @@
+function getLetter(s) {
+    let letter;
+
+    switch(s[0])
+    {
+        case 'a':case 'e':case 'i':case 'o': case 'u': letter= 'A'; break;
```

Comment:
these are too many cases without any strong logic. Use some conditional checks and use only limited cases. too many cases indicate this is not clean design

Reply:
how can we do that

Reply:
write like below : 

switch(s[0]){

case ('a' && 'e' && 'i' && 'o' && 'u') : letter = 'A' ; break;

Reply:
it is not working

Reply:
`function getLetter(s) {
    let letter;

    switch(s[0])
    {
        case ('a' || 'e' || 'i' || 'o' || 'u') : letter = 'A' ; break;
        case ('b' || 'c' || 'd' || 'f' || 'g'): letter= 'B'; break;
        case ('h' || 'j' || 'k' || 'l' || 'm'): letter= 'C'; break;
        default: letter = 'D';
    }
    
    return letter;
}; getLetter('Apple');
"D"
getLetter('apple');
"A"`